src/App/App.js

```diff
@@ -67,8 +69,8 @@ class App extends Component {
 
   updateLoginView = () => {
     if (!this.state.user.name) {
-      this.setState({hasLoginView: true}) 
-    } 
+      this.setState({hasLoginView: true})
```

Comment:
The logic still seems a little off here, we could look into setting `hasLoginView` in a different place, just not sure where yet.